packages/liquidator/src/liquidator.js

```diff
@@ -407,22 +421,22 @@ class Liquidator {
       this.logger.debug({
         at: "Liquidator",
         message: "Detected a pending or expired liquidation",
-        liquidation: JSON.stringify(liquidation)
+        liquidation: liquidation
       });
 
       // Construct transaction.
       const withdraw = this.empContract.methods.withdrawLiquidation(liquidation.id, liquidation.sponsor);
 
       // Confirm that liquidation has eligible rewards to be withdrawn.
-      let withdrawAmount, gasEstimation;
```

Comment:
between version 1.2.2 and the latest, we changed how withdrawals work. now, this _either_ returns the amount withdrawn _or_ returns an object containing extra info (such as other positions withdrawn).